### PR TITLE
Make partition count and consumer group optional params for longhaul …

### PR DESCRIPTION
**Pull request by ewertons**
…tests

 

# Checklist
- [x] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-c/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.
- I submitted this PR against the correct branch: 
  - [x] This pull-request is submitted against the `master` branch. 
  - [ ] I have merged the latest `master` branch prior to submission and re-merged as needed after I took any feedback.
  - [ ] I have squashed my changes into one with a clear description of the change.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c [hunk 1]
```diff
@@ -51,6 +51,9 @@
 #define MAX_TWIN_DESIRED_PROP_TRAVEL_TIME_SECS  300.0
 #define MAX_TWIN_REPORTED_PROP_TRAVEL_TIME_SECS 300.0
 
+#define DEFAULT_CONSUMER_GROUP         "$Default"
+#define DEFAULT_PARTITION_COUNT        16
```
Comment (jebrando) [inline iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c]: I believe by default IotHubs create the Event hubs with 4 partitions.

**Comment by ewertons** [comment]
I need to review the fix, will close this for now.

**Pull request closed without merging**